pkg/printers/internalversion/printers.go

```diff
@@ -201,6 +204,10 @@ func AddHandlers(h printers.PrintHandler) {
 	h.Handler(statusColumns, nil, printStatus)
 	h.Handler(controllerRevisionColumns, nil, printControllerRevision)
 	h.Handler(controllerRevisionColumns, nil, printControllerRevisionList)
+	h.Handler(initializerConfigurationColumns, nil, printInitializerConfiguration)
```

Comment:
Please add a new style printer (table) rather than old style.  See other examples in this file